website/pages/en/playground.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @format
+ */
+
+const CompLibrary = require('../../core/CompLibrary.js');
```

Comment:
This could be:
```js
  const { Container } = require('../../core/CompLibrary.js');
``

But I'm fine with the current style as that appears to match the rest of the codebase. However, because the website codebase is relatively tiny, we shouldn't be too formal in following its style, since it's not a big effort to upgrade the rest of the codebase to a more modern style if we so wish.

(Though these will be changed again when moving to Docusaurus 2 and ES6 imports, I suppose...)

Reply:
That make sense. I thought it would be a good idea to follow the same the exact same style, wasn't sure if it could be a bit flexible.